src/main/java/core/basesyntax/services/Obtainable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.services;
+
+public interface Obtainable {
```

Comment:
Obtainable mean that something could be obtained, and logically there should be method 'obtain'. Think about renaming